clients/src/main/java/org/apache/kafka/clients/consumer/internals/StreamsRebalanceData.java

```diff
@@ -346,11 +349,24 @@ public Map<HostInfo, List<TopicPartition>> partitionsByHost() {
         return partitionsByHost.get();
     }
 
+    /** For the current stream thread to request a shutdown of all Streams clients belonging to the same applications. */
```

Comment:
The comment of `shutdownRequested` uses singular. It would probably be better to unify it.
```suggestion
    /** For the current stream thread to request a shutdown of all Streams clients belonging to the same application. */
```